### ISSUE TITLE
Support different response times for different public bodies (WDTK-67)

### ISSUE THREAD

**Issue by oajirabot** [openaustralia/righttoknow#73]
<p>We are only launching with support for federal public bodies. Before we can support states we need Alaveteli to support different response time calculations based on what jurisdiction the public bodies is in.</p>

<p>See discussion on <a href="/browse/WDTK-43" title="Check statutory times for responses under Australian law and update our configuration"><del>WDTK-43</del></a>.</p>

---

**Issue details imported from Jira:**

| Detail | Value |
| --- | --- |
| Assignee | Unassigned |
| Reporter | Henare Degan |
| Created | 2012-09-23 13:41:49 |
| Updated | 2012-09-23 13:47:40 |

---
### Imported Jira Comments

**Henare Degan** - 2012-09-23 13:47:40

> <p>I've started the discussion with upstream: <a href="https://groups.google.com/forum/?fromgroups=#!topic/alaveteli-dev/OHgWUkTpe0c" class="external-link">https://groups.google.com/forum/?fromgroups=#!topic/alaveteli-dev/OHgWUkTpe0c</a></p>
>

**Comment by benrfairless** [comment]
I don't think we should work rigidly to the timeframes, we should provide a general guide. Closing as wontfix

**Comment by mlandauer** [comment]
I think having timeframes in place is very helpful for the user

**Comment by henare** [comment]
#450 overrides the calculation of request due date.

It doesn't affect the calculation of the "very overdue" date, which we currently have set to 60 days. Given that the longest due date in Australia is 45 days (VIC & WA), I think it's still reasonable to leave it at that.

There's one other place where we need to make a change for this to be complete - the flash message shown immediately after we send a request that says you should get a response in X days.

In the version of Alaveteli we're currently running it's [set in the controller](https://github.com/openaustralia/alaveteli/blob/6d5d1f5618c4c7d297b0373941336f0603635593/app/controllers/request_controller.rb#L364-L369) but happily this was [changed recently](https://github.com/mysociety/alaveteli/pull/2078) and is now extracted into a partial. This means we can easily [override it](https://github.com/mysociety/alaveteli/blob/8c4e401f7a7f5ac4149f007cd0cf36968045d329/app/controllers/request_controller.rb#L375t) in our theme too.
